StartEmailValidationProcess/handler.ts

```diff
@@ -99,13 +107,32 @@ export function StartEmailValidationProcessHandler(
     );
 
     const dfClient = df.getClient(context);
-    await dfClient.startNew(
-      "EmailValidationProcessOrchestrator",
-      undefined,
-      emailValidationProcessOrchestartorInput
-    );
-
-    return ResponseSuccessAccepted();
+    return taskEither
+      .of(makeStartEmailValidationProcessOrchestratorId(fiscalCode, email))
+      .chain(orchId =>
+        isOrchestratorRunning(dfClient, orchId).foldTaskEither<
+          Error,
+          IResponseSuccessAccepted
+        >(fromLeft, _ =>
```

Comment:
No need to cast the left type, we can just use chain here:

```suggestion
        isOrchestratorRunning(dfClient, orchId).chain(_ =>
```

Reply:
Done 👍